logic/src/commonTest/kotlin/com/wire/kalium/logic/data/user/UserMapperTest.kt

```diff
@@ -38,21 +38,21 @@ import kotlin.test.assertEquals
 
 class UserMapperTest {
 
-    @Test
-    fun givenUserProfileDTOAndUserTypeEntity_whenMappingFromApiResponse_thenDaoModelIsReturned() = runTest {
-        // Given
-        val givenResponse = TestUser.USER_PROFILE_DTO
-        val givenUserTypeEntity = UserTypeEntity.EXTERNAL
-        val expectedResult = TestUser.ENTITY.copy(
-            phone = null, // UserProfileDTO doesn't contain the phone
-            connectionStatus = ConnectionEntity.State.NOT_CONNECTED // UserProfileDTO doesn't contain the connection status
-        )
-        val (_, userMapper) = Arrangement().arrange()
-        // When
-        val result = userMapper.fromApiModelWithUserTypeEntityToDaoModel(givenResponse, givenUserTypeEntity)
-        // Then
-        assertEquals(expectedResult, result)
-    }
+//     @Test
+//     fun givenUserProfileDTOAndUserTypeEntity_whenMappingFromApiResponse_thenDaoModelIsReturned() = runTest {
+//         // Given
+//         val givenResponse = TestUser.USER_PROFILE_DTO
+//         val givenUserTypeEntity = UserTypeEntity.EXTERNAL
+//         val expectedResult = TestUser.ENTITY.copy(
+//             phone = null, // UserProfileDTO doesn't contain the phone
+//             connectionStatus = ConnectionEntity.State.NOT_CONNECTED // UserProfileDTO doesn't contain the connection status
+//         )
+//         val (_, userMapper) = Arrangement().arrange()
+//         // When
+//         val result = userMapper.fromApiModelWithUserTypeEntityToDaoModel(givenResponse, givenUserTypeEntity)
+//         // Then
+//         assertEquals(expectedResult, result)
+//     }
```

Comment:
Is this intended?

Reply:
I forgot to delete it after deleting this mapping function.

Reply:
Decided to resurrect it since test coverage of the other function was completely missing.